composer.json

```diff
@@ -0,0 +1,27 @@
+{
+	"name": "qa-tools/behat-example",
+	"license": "BSD-3-Clause",
+	"require": {
+		"php": ">=5.3.1",
+		"qa-tools/qa-tools": "~1.0@dev",
+		"qa-tools/behat-extension": "dev-cleanup",
+		"mindplay/annotations": "~1.2@dev",
+		"behat/mink": "~1.5@dev",
+		"behat/mink-extension": "*@dev",
+		"behat/mink-selenium2-driver": "~1.1@dev"
+	},
+	"repositories": [
+		{
+			"type": "vcs",
+			"url": "/Users/michi/Documents/Workspace/behat-extension"
+		}
+	],
+	"autoload-dev": {
+		"psr-0": {
+			"QATools\\Example": "./src"
+		}
+	},
+	"config": {
+		"bin-dir": "bin/"
```

Comment:
The `bin-dir` is specified for a repo, when it offers any binaries to end user, so that Composer will know what to symlink to the `/vendor/bin` folder.